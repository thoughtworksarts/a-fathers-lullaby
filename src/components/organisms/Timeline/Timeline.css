.Timeline {
    margin-top: 50px;
    padding-top: 10px;
    display: flex;
    justify-content: center;
    flex-direction: column;
    background-color: white;
    width: 70vw;
}