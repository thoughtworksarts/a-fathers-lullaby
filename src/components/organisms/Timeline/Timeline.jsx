import React from 'react'
import './Timeline.css'
import { TimelineEntry } from 'organisms'
import * as myConstClass from '../../atoms/TimelineDescriptions/TimelineDescriptions.jsx'
import picOne from 'assets/timelinepic_one.png'
import picTwo from 'assets/timelinepic_2018_1.png'
import picThree from 'assets/timelinepic_2018_2.png'

const Timeline = () => {
  return (
    <div className='Timeline'>
      <TimelineEntry year='2018' description={myConstClass.par2018Desc2} pictureFirst='true' picture={picThree} altText='People at audio installation' />
      <TimelineEntry year='2018' description={myConstClass.par2018Desc1} pictureFirst='false' picture={picTwo} altText='People sitting and listening to audio' />
      <TimelineEntry year='2017' description={myConstClass.par2017} pictureFirst='true' picture={picOne} altText='Woman singing lullaby' />
    </div>
  )
}

export default Timeline
