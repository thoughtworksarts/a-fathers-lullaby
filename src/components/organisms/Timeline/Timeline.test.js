import React from 'react'
import Timeline from './Timeline'
import { shallow } from 'enzyme'

describe('Timeline', () => {
  it('matches snapshot', () => {
    const timelineComponent = shallow(<Timeline />)
    expect(timelineComponent).toMatchSnapshot()
  })
})
