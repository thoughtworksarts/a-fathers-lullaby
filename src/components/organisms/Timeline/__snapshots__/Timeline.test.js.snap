// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timeline matches snapshot 1`] = `
<div
  className="Timeline"
>
  <TimelineEntry
    entryNum="a-fathers-lullaby/about/3"
    pictureFirst="true"
    timelineInfo={
      Object {
        "alt_text": "People at audio installation",
        "full_desc": "At 2018 HUBweek, We The Future conference, we prototyped an immersive participatory installation inside a shipping container; a self-contained model of a complex interactive media installation that could be accessible at any public site as a low budget traveling exhibit, alternative to a gallery or museum space. Collaborators: Hisham Bedri (pressure sensor panels with audio playback and light activation), Christian Gentry (live sound composition), Halsey Burgen (geolocated participatory website), Farid Manshadi & Jason Bashaw & Omid Fallahazad & Nedallah Fahandej & Kevin Long (installation), Afshin Fahandezh & Lizandro Segura (documentation)",
        "img": "timelinepic_2018_2.png",
        "short_desc": "At 2018 HUBweek, We The Future conference, we prototyped an immersive participatory installation inside a shipping container. ",
        "year": "2018",
      }
    }
  />
  <TimelineEntry
    entryNum="a-fathers-lullaby/about/2"
    pictureFirst="false"
    timelineInfo={
      Object {
        "alt_text": "People sitting and listening to audio",
        "full_desc": "In the second chapter, 2018, the project expanded through collaboration with local artists and partnership with institutions, such as the Federal Probation Office, the newly formed Office of Returning Citizens and commissioned by the Boston Center for the Arts.The result was a site - responsive, public sound installation on Tremont St., Boston Center for the Arts public plaza, July - Oct 2018.",
        "img": "timelinepic_2018_1.png",
        "short_desc": "In the second chapter, 2018, the project expanded through collaboration with local artists and partnership with institutions, such as the Federal Probation Office. ",
        "year": "2018",
      }
    }
  />
  <TimelineEntry
    entryNum="a-fathers-lullaby/about/1"
    pictureFirst="true"
    timelineInfo={
      Object {
        "alt_text": "Woman singing lullaby",
        "full_desc": "The first chapter of the project was launched in 2017, when artist served as Boston Artist-in-Residence with mayor’s Office of Art and Culture and Boston Centers for Youth & Families, in collaboration with Blackstone Community Center in South End and in connection with my role as Research Fellow at MIT Open Documentary Lab.
The culmination of these experiences was shared with the public in a participatory media installation at Villa Victoria Center for the Arts, 2017, at the heart of the community where the work was produced.",
        "img": "timelinepic_one.png",
        "short_desc": "The first chapter of the project was launched in 2017, when artist served as Boston Artist-in-Residence with mayor’s Office of Art and Culture and Boston Centers for Youth & Families. ",
        "year": "2017",
      }
    }
  />
</div>
`;
